src/amo/components/ServerError/index.js

```diff
@@ -0,0 +1,71 @@
+import React, { PropTypes } from 'react';
+import { oneLine } from 'common-tags';
+import { compose } from 'redux';
+import NestedStatus from 'react-nested-status';
+
+import Link from 'amo/components/Link';
+import { ADDON_TYPE_EXTENSION, ADDON_TYPE_THEME } from 'core/constants';
+import translate from 'core/i18n/translate';
+import { sanitizeHTML, visibleAddonType } from 'core/utils';
+import Card from 'ui/components/Card';
+
+import 'amo/components/ErrorPage/ErrorPage.scss';
```

Comment:
Yeah, I actually have that ready in a separate patch that builds on this PR (it also adds generic errors for disco pane and that can be used anywhere). But it's an extra few hundred lines and I wanted to keep this PR small. I have done exactly what you suggested in https://github.com/mozilla/addons-frontend/tree/generic-error-page; I'll make a PR with it after this if that's okay.